src/applications/disability-benefits/686c-674/config/helpers.js

```diff
@@ -34,7 +34,7 @@ export const VaFileNumberMissingAlert = (
       number.
     </p>
     <p className="vads-u-font-size--base">
-      You'll need to update your personal information. Please call Veterans
```

Comment:
Looks like this was from the new eslint rule?

Reply:
Sorry that this bled into your PR!